ablation_eval.py

```diff
@@ -254,6 +306,7 @@ def create_job_configs(
                     "model": model_kwargs,
                     "tokenizer_name": main_config.tokenizer_name,
                     "scheduler": main_config.scheduler,
+                    "optimizer": task_config.get("optimizer", main_config.get("optimizer", None)),
```

Comment:
This line errors out when running the tests: `AttributeError: 'NoneType' object has no attribute 'get'`

Reply:
Done.